### PR TITLE
PageManager added. Formatting done.

### PR DESCRIPTION
**Pull request by Davit-Sim**
1) PageManager added to reduce code repeatability. Also readability improved (more human like code).
2) Describe blocks added for better test separation.
3) General formatting has been applied across files to increase readability and maintainability

Test run:
All tests still run and pass after changes done.

image

### CODE REVIEW

File: pageObjects/signInPage.ts [hunk 1]
```diff
@@ -1,7 +1,6 @@
-import {Page} from "@playwright/test"
+import { Page } from "@playwright/test"
 
-export class SignInPage
-{
+export class SignInPage{
```
Comment (eliskavo) [inline pageObjects/signInPage.ts:3]: Add a space

File: pageObjects/signInPage.ts [hunk 2]
```diff
@@ -1,7 +1,6 @@
...
     readonly page: Page
 
     constructor(page: Page){
```
Comment (eliskavo) [inline pageObjects/signInPage.ts:6]: add a space

File: pageObjects/pageManager.ts [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  }
+
+  onJoinBukKlabPage() {
+    return this.joinBukKlabPage;
+  }
+
+  onSignInPage() {
+    return this.signInPage;
+  }
+
```
Comment (eliskavo) [inline pageObjects/pageManager.ts:58]: delete this extra line

**Review by eliskavo — APPROVED** [review]

File: pageObjects/signInPage.ts (hunk 2 above)
Comment (Davit-Sim) [reply pageObjects/signInPage.ts]: Done

File: pageObjects/signInPage.ts (hunk 1 above)
Comment (Davit-Sim) [reply pageObjects/signInPage.ts]: Done

File: pageObjects/pageManager.ts (hunk 3 above)
Comment (Davit-Sim) [reply pageObjects/pageManager.ts]: Done

**Pull request merged**